File: src/e1palindromepermutation/Python/main.py
# main.py

from palindrome_permutation import can_permute_palindrome

test_cases = [
    "code", "aab", "carerac", "racecar", "hello",
    "baefeab", "abc", "xzz", "jjadd", "kllk"
]

for test_case in test_cases:
    result = can_permute_palindrome(test_case)
    print(f'Can "{test_case}" form a palindrome permutation? {result}')

File: src/e1palindromepermutation/JS/main.js
// main.js

const canPermutePalindrome = require('./palindromePermutation');

const testCases = [
    "code", "aab", "carerac", "racecar", "hello",
    "baefeab", "abc", "xzz", "jjadd", "kllk"
];

testCases.forEach(testCase => {
    console.log(`Can "${testCase}" form a palindrome permutation? ${canPermutePalindrome(testCase)}`);
});
